modules/react/button/lib/PrimaryButton.tsx

```diff
@@ -123,7 +118,7 @@ export const PrimaryButton = createComponent('button')({
     {
       size = 'medium',
       iconPosition = 'start',
-      variant,
+      variant = undefined,
```

Comment:
I'm curious why this `= undefined` needs to be here (apologies if this has already been covered elsewhere).

Reply:
It isn't needed. I'll remove it.